pkg/filters/filters_test.go

```diff
@@ -205,5 +206,7 @@ func TestFiltersFromRequest(t *testing.T) {
 	expectedLibpodFilters := []string{"label=xyz=bar", "label=abc", "reference=test"}
 	got, err := FiltersFromRequest(&req)
 	require.NoError(t, err)
+	sort.Strings(expectedLibpodFilters)
+	sort.Strings(got)
```

Comment:
I think you can use `assert.ElementsMatch()` instead of sorting explicitly

Reply:
Ooh, that's a good one.  I missed that somehow when doing a search of the functions in the package.  Changing it.